### PR TITLE
Api to db

### PR DESCRIPTION
**Pull request by EtiWitty**
Please review my PR.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,20 +1,63 @@
...
+
+app.get('/attributesrating/:attribute?', (req, res) => {
+  db.getRatingAttributes((err, data) => {
+    if (err) {
+      res.status(500);
+      res.send(err);
+    } else {
+      res.status(200);
+      let result;
+      if (req.params.attribute) {
```
Comment (YesImKenny) [inline server/index.js]: I like how you handled this condition to get the correct data.

File: server/db/controller.js [hunk 2]
```diff
@@ -1,10 +1,40 @@
+/* eslint-disable object-shorthand */
+/* eslint-disable func-names */
 
 const connection = require('./index.js');
-const getAll = function(cb){
-    // execute my query to the db
-    connection.query('SELECT * FROM reviews', cb);
-}
+
+const getReviewItems = function (cb) {
```
Comment (YesImKenny) [inline server/db/controller.js]: Is this query returning every single review in your database? If so, you may want to return only reviews pertaining to one specific listing. Just something to consider.

File: server/index.js (hunk 1 above)
Comment (EtiWitty) [reply server/index.js]: Thank you!

File: server/db/controller.js (hunk 2 above)
Comment (EtiWitty) [reply server/db/controller.js]: Well, I have listing and property ids, which I haven't used them for now to keep the current scope limited.But I will include it to all my queries later.

**Pull request merged**